src/ui/veles_mainwindow.cc

```diff
@@ -647,6 +702,21 @@ QTabBar* MainWindowWithDetachableDockWidgets::dockWidgetToTab(
   return nullptr;
 }
 
+void MainWindowWithDetachableDockWidgets::splitDockWidget2(
+    QDockWidget* first, QDockWidget* second, Qt::Orientation orientation) {
+#ifdef SPLIT_DOCK_WIDGET_ACTUALLY_WORKS_AS_ONE_COULD_EXPECT
```

Comment:
Then unless you plan on defining it, I don't think it should be an ifdef - it creates confusion that sometimes the behaviour differs, while it is just a rant that could be contained to comment.